spring-kafka/src/test/java/org/springframework/kafka/support/micrometer/ObservationTests.java

```diff
@@ -387,6 +393,41 @@ void observationErrorException(@Autowired ExceptionListener listener, @Autowired
 				.hasMessage("obs5 error");
 	}
 
+	@Test
+	void observationErrorExceptionWhenCompletableFutureReturned(@Autowired ExceptionListener listener, @Autowired SimpleTracer tracer,
+			@Autowired @Qualifier("throwableTemplate") KafkaTemplate<Integer, String> errorTemplate,
+			@Autowired KafkaListenerEndpointRegistry endpointRegistry)
+			throws ExecutionException, InterruptedException, TimeoutException {
```

Comment:
For readability I prefer a blank line after such a long method declaration.
Like you do in the other new test.
Thanks